Sources/CryptoBoringWrapper/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ target_include_directories(CryptoBoringWrapper PUBLIC
   $<TARGET_PROPERTY:CCryptoBoringSSLShims,INCLUDE_DIRECTORIES>)
 
 target_link_libraries(CryptoBoringWrapper PUBLIC
+  $<$<NOT:$<PLATFORM_ID:Darwin>>:Foundation>
```

Comment:
@compnerd Am I correct to think that this should fix the error we see in SwiftPM Windows build?

Also, do we need `$<$<NOT:$<PLATFORM_ID:Darwin>>:dispatch>` here too? We have it in https://github.com/apple/swift-crypto/blob/main/Sources/Crypto/CMakeLists.txt#L91

Reply:
Do you have a pointer to the logs for the Windows build that I should be looking at?  But, this should be safe to link on Windows.  The only reason that we do not link against it on Darwin is that swift-corelibs-foundation is renamed to SwiftFoundation, which makes this a lot more complicated.  Instead, we are relying on the implicit linking to Foundation on the system for this.

Reply:
Update: found the logs!  Yes, that seems like it _should_ fix the Windows error.  Should kick off a test with cross-repository testing ideally.

Reply:
Right, if you look at the command invocation, there is no search path to Foundation's swiftmodule.  The `target_link_libraries` not only ensures that we are correctly linking the dependencies but also propagates the include paths so that the swiftmodule is searched for in the correct location.

Reply:
@compnerd https://ci-external.swift.org/job/swiftpm-PR-windows/635/ passes with this change https://github.com/apple/swift-package-manager/pull/5920